frontend/src/Components/loginForm.tsx

```diff
@@ -35,6 +39,7 @@ export const LoginForm = () => {
           {/* T80 (FK): Add route to sign up page once sign up code is push and merged */}
           <p className='subheader'>Don't have an account?<a href='' > Sign up</a> </p>
           <form action="/login/password" method="post" onSubmit={handleSubmit}>
+              <p className='error_message'>{errorMessage}</p>
```

Comment:
a common tactic here is to conditionally render the whole element, rather than leaving this as is

```
{errorMessage && <p className='error_message'>{errorMessage}</p>}
```

reason being that even if errorMessage is an empty string, the `p` element might still have styling that affects the rest of the page

Reply:
Okay I see. Changed!